plugins/foreman_remote_execution/0.0/index.md

```diff
@@ -0,0 +1,373 @@
+
+<table class="table table-bordered table-condensed">
+  <tr>
+    <th>Plugin</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>foreman_remote_execution</td>
+    <td>Required, contains the UI part and currently also SSH provider, depends on foreman-tasks</td>
+  </tr>
+  <tr>
+    <td>foreman_tasks</td>
+    <td>Required, keep status of all executed commands, allows debugging if something goes wrong</td>
+  </tr>
+  <tr>
+    <td>smart_proxy_remote_execution_ssh</td>
+    <td>Smart proxy plugin, required for using SSH provider, proxies the communication between Foreman and hosts, requires smart_proxy_dynflow</td>
+  </tr>
+  <tr>
+    <td>smart_proxy_dynflow</td>
+    <td>Smart proxy plugin, runs the jobs asynchronously and keeps status of commands executed via this proxy</td>
+  </tr>
+</table>
+
+Each component is packaged for platforms that Foreman is officially
+packaged for. Since both Foreman a Smart proxy plugins requires
+Dynflow, Ruby 1.9+ is required. Foreman 1.9+ should be compatible with
+0.0.* versions.
+
+# 2. Installation
+
+This chapter covers clean Foreman and plugins installation from
+nightly repositories on CentOS 7.1. We assume smart proxy being
+installed on the same host. Please adapt according to your needs.
+
+Start by installing Foreman nightly repository and EPEL7 by
+
+    yum install http://yum.theforeman.org/nightly/el7/x86_64/foreman-release.rpm
+    yum install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+
+Install the foreman-installer
+
+    yum install foreman-installer
+
+Run foreman-installer and let it install foreman-tasks plugin
+
+    foreman-installer --enable-foreman-plugin-tasks
+
+After it sucessfully finishes you'll have to manually install plugins
+by following command
+
+    yum install ruby193-rubygem-foreman_remote_execution rubygem-smart_proxy_remote_execution_ssh
+
+Now migrate the database and load default data
+
+    foreman-rake db:migrate
+    foreman-rake db:seed
+
+Next you have to setup ssh keys. By default smart proxy loads the key
+from `/usr/share/foreman-proxy/.ssh/id_rsa_foreman_proxy` and use root
+user on target host. To customize it you can edit the configuration in
+`/etc/foreman-proxy/config/settings.d/dynflow.yml`. Without
+customization you need to create new ssh key and distribute it to
+target hosts. The key must not use passphrase.
+
+To generate a key, run following command on the host where Smart proxy runs
+
+    mkdir ~foreman-proxy/.ssh
+    chown foreman-proxy ~foreman-proxy/.ssh
+    sudo -u foreman-proxy ssh-keygen -f ~foreman-proxy/.ssh/id_rsa_foreman_proxy -N ''
+
+Don't forget to restart Foreman, Smart proxy and Foreman tasks so
+plugins are loaded
+
+    service httpd restart
+    service foreman-tasks restart
+    service foreman-proxy restart
+
+Finally, you have to refresh the Smart proxy features in the Foreman:
+Dynflow and Ssh should appear there as new features of the Smart proxy.
```

Comment:
a feature is called `Ssh` and it will appear this way in UI, I opened an issue to rename it first http://projects.theforeman.org/issues/11406